experiment_scheduler/task_manager/task_manager_server.py

```diff
@@ -129,22 +129,26 @@ def use_gpu(self):
         """
         return self._use_gpu
 
+    @start_end_logger
     def health_check(self, request, context):
         """
         Return current server status and finished task id list which are finished
         """
+
         server_status = ServerStatus()
         server_status.alive = True
+        server_status.task_status_array.extend([])
```

Comment:
Seems like extend does nothing in this line.

Reply:
this line is necessary for appending task status